code/modules/reagents/chemistry/reagents.dm

```diff
@@ -82,7 +82,7 @@
 	var/purge_multiplier = 1
 
 	///The default reagent container for the reagent, used for icon generation
-	var/obj/item/reagent_containers/default_container = /obj/item/reagent_containers/cup/bottle
+	var/obj/default_container = /obj/item/reagent_containers/cup/bottle
```

Comment:
how is this not guarenteeable

Reply:
`default_container = /obj/effect/decal/cleanable/ash`
`default_container = /obj/effect/decal/cleanable/oil`
`default_container = /obj/effect/decal/cleanable/greenglow`
literally how could it ever be guaranteeable with this